doc/storage/appendix/sra.rst

```diff
@@ -0,0 +1,919 @@
+      :impact: VH
+      :likelihood: VH
+
+   .. residual:: DM.EXPOSED
+      :impact: VH
+      :likelihood: VL
+      
+   .. unmitigated:: DM.AUTHORISED
+      :impact: VH
+      :likelihood: H
+
+   .. residual:: DM.AUTHORISED
+      :impact: VH
+      :likelihood: VL
+
+   .. unmitigated:: DM.SECURE_CHANNEL
+      :impact: VH
+      :likelihood: H
+
+   .. residual:: DM.SECURE_CHANNEL
+      :impact: VH
+      :likelihood: VL
+
+
+.. threat:: Man-in-the-middle
+   :id: MITM
+
+   .. description:: 
+      An attacker can interfere with communication and replace the transmitted data. In this threat we only consider attackers between the Storage Service and the Storage Medium. An attacker interposing between the Caller and the Storage Service is considered under `T.SPOOFREAD` or `T.SPOOFWRITE`. 
+      
+      For `DM.PROTECTED` the Storage Service and the Storage Medium are within the :term:`PRoT` partition and therefore isolated. 
+     
+      For `DM.EXPOSED` any code running in the :term:`NSPE` may be able to access the Storage Medium and any driver firmware, and therefore act as a man-in-the-middle, by for example persuading the Storage Service to write to one buffer, and the Storage Medium to read from another. Therefore, the data must be properly encrypted for confidentiality and protected with a MAC, or signature, for integrity. 
+      
+      For `DM.AUTHORISED` while a man-in the middle can eavesdrop on data in transit, all commands need to be authenticated, and all replies include some verification. By definition, the Storage Medium verifies all commands. Therefore, provided the Storage Service verifies all replies, including checking the status codes returned after it submits a write command, before releasing any data to their caller, the man-in-the-middle cannot change the command or reply without being detected.
+      
+      For `DM.SECURE_CHANNEL` if the secure channel is set up without authorisation, it would be vulnerable to a man-in-the-middle attack. Given that in this implementation we are not expecting encryption of the data - other than by the channel itself - the attacker would have full access to stored data. 
+
+   .. mitigations:: 
+      `M.Encrypt` Encryption ensures that the Man-in-the-middle does not know what data is being transferred. It also means they cannot force a specific value to be stored.  
+    
+      :mitigation:`MAC` In `DM.EXPOSED`, applying a Message Authentication Code or a signature or using an authenticated encryption scheme means that the Storage Service can check the integrity of the data when it is read back from the Storage Medium.
+      
+      :mitigation:`UniqueKeys` For `DM.AUTHORISED` and `DM.SECURE_CHANNEL`, requiring keys used by the storage partition to be unique to each instance ensures that an attacker can only learn the key used on this instance on this specific instance. They cannot construct a class break by discovering the key for a single instance. 

```

Comment:
Clarify 'instance' - presumably this means at least 'device instance', but could mean 'storage service instance', if there was more than one on a single device?